### DISCUSSION TITLE
MFN field for convertible notes

### DISCUSSION THREAD

**Discussion by magonjr** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#235] [Validation]
We propose adding a MFN field for convertible notes. We noticed that OCF only has a Most Favored Nations Date field for SAFEs, but not for convertible notes. While it’s a pretty rare mechanic for convertible notes, it is possible for convertible notes to have MFN clauses.

**Comment by JSv4** [comment]
See issue #250
